src/main/java/CustomThreadImplementingRunnable.java

```diff
@@ -0,0 +1,15 @@
+import org.apache.log4j.Logger;
+
+public class CustomThreadImplementingRunnable implements Runnable {
+    private static final Logger LOGGER = Logger.getLogger(CustomThreadImplementingRunnable.class);
+    private int value;
```

Comment:
```suggestion
    private int value;
```
Let’s create class Counter with field and methods. Use one object of counter in two threads, not separate for each thread.